### PR TITLE
refactor and fixes issue #13

### LINKED ISSUE #13: Input Fields Are Too Small
**Describe the bug**
The input fields are too small

**Expected behavior**
The user expects a larger input field for easier use.

**Desktop (please complete the following information):**
 - OS: ALL
 - Browser: ALL
 - Versions: ALL

**Smartphone (please complete the following information):**
I have not tested this on a smartphone, but iPad screens also show that the input is too small.

**Additional context**
This is probably a simple CSS fix.

### PR DESCRIPTION
**Pull request by xNocken**
adds mobile version, makes inputs to number inputs, makes size of inputs bigger, changed value to placeholders fixes #13

### CODE REVIEW

**Comment by cdrani** [comment]
@xNocken : Job well done! Thanks for changing replacing the `value` attribute with the `placeholder` one for the text input elements! I see that you have included the issue number in the title of the pull request (PR), which is great, but it would be stupendous if you start referencing the issue in the PR description as well. https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords

**Comment by xNocken** [comment]
@cdrani the #13 is the issue. I dont know how to mark it right lol.

File: index.html [hunk 1]
```diff
@@ -13,11 +13,11 @@
 <body>
   <h1>Welcome to The Shade Generator!</h1>
 
-  <input type="text" class="red" value="Red Value"></input>
+  <input type="number"  max="255" min="0"class="red" placeholder="Red Value"></input>
```
Comment (varlevi) [inline index.html:17]: What are the max and min values for? Do they change how the JavaScript, or are they just to make the code easier to read?

Comment (varlevi) [reply index.html:17]: Oh. I see. They create the Up and Down buttons on the side. Great addition! Is there any way to make them match the website theme a little more? I feel that they stick out a little.

File: css/styles.css [hunk 2]
```diff
@@ -140,3 +141,30 @@ footer {
...
   border-radius: .75em;
 }
+
+@media screen and (max-width: 992px) {
+  h1 {
+    font-size: 6em;
+    margin: 60px 0 40px;
+  }
+  
+  input {
```
Comment (varlevi) [inline css/styles.css:151]: The inputs still seem too big on mobile devices. Also, the margin between them is a little small.

File: css/styles.css [hunk 3]
```diff
@@ -140,3 +141,30 @@ footer {
   border: solid 2px red;
   border-radius: .75em;
 }
+
```
Comment (varlevi) [inline css/styles.css:144]: Thank you for keeping mobile users in mind! This is a great improvement.

**Comment by varlevi** [comment]
Thank you for contributing! These improvements are really helpful. As you have probably seen, I had a few questions, but once these are resolved I'll merge these.

File: index.html (hunk 1 above)
Comment (xNocken) [reply index.html:17]: I'm not that good at css. I Googled but haven't found anything.

**Comment by cdrani** [comment]
@xNocken : You would add something like:`Closes #13` or `Fixes #13` in the PR description at the end, which is the first comment you had:
> adds mobile version, makes inputs to number inputs, makes size of inputs bigger, changed value to placeholders

`Closes` and `Fixes` are keywords for github to mark the PR as closed when it has been merged.

File: css/styles.css (hunk 2 above)
Comment (xNocken) [reply css/styles.css:151]: changed that in my new pr. If it still doesnt match then please say how you would like to have it

**Comment by varlevi** [comment]
I have implemented this feature, but it still in beta, so please help check for bugs!

**Pull request merged**